src/components/Inventory/MyInventory.js

```diff
@@ -77,6 +79,8 @@ function MyInventory() {
 
   const classes = useStyles();
   return (
+    <Grid container justify="center" spacing={0}>
+        <Navbar/>
```

Comment:
want to add maybe a grid item around the navbbar, or else the grid container is pretty useless